Makefile

```diff
@@ -15,17 +15,8 @@
 # Args:
 # GCE_PD_CSI_STAGING_IMAGE: Staging image repository
 REV=$(shell git describe --long --tags --match='v*' --dirty 2>/dev/null || git rev-list -n1 HEAD)
-ifdef GCE_PD_CSI_STAGING_VERSION
-	STAGINGVERSION=${GCE_PD_CSI_STAGING_VERSION}
-else
-	STAGINGVERSION=${REV}
-endif
-
-GCFLAGS=""
```

Comment:
GCFLAGS was removed too, is this ok?

Reply:
They aren't used anymore, but this will allow them to be set from an env. That was my understanding at any rate.

Reply:
the value of GCFLAGS was set conditionally based on this flag before:

```
GCFLAGS=""
ifdef GCE_PD_CSI_DEBUG
	GCFLAGS="all=-N -l"
endif
```

`GCE_PD_CSI_DEBUG` is set in `Dockerfile.debug`

Reply:
ohh, my fault! i might remove that by mistake :(